src/view-data/method.ts

```diff
@@ -113,9 +113,10 @@ function getPathToMethodName(httpVerb: string, path: string): string {
   });
 
   const result = camelCase(segments.join("-"));
-  return `${httpVerb.toLowerCase()}${result[0].toUpperCase()}${result.substring(
-    1
-  )}`;
+  if(result.length > 0)
```

Comment:
Hm, were these changes done directly in the github editor? Seems like prettier hasnt run, so the formatting is off, as the prettified formatting should look something like:
````
if (result.length > 0) {
    return `${httpVerb.toLowerCase()}${result[0].toUpperCase()}${result.substring(1)}`;
} else {
    return "root";
}
````